CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ CHANGELOG
 unreleased
 ----------
 - Update braintree-web to v3.76.4
+- Add `threeDSecure.cardinalSDKConfig` option to `dropin.create`
```

Comment:
```suggestion
- Add `threeDSecureOptions.cardinalSDKConfig` option to `dropin.create`
```

Reply:
It's:

```
braintree.dropin.create({
  threeDSecure: {
    cardinalSDKConfig: { /* options */ }
  }
```

`threeDSecureOptions` is a type definition in the JSDOC, so we can link to it in the options from `create`. See the `threeDSecure` option here: https://braintree.github.io/braintree-web-drop-in/docs/current/module-braintree-web-drop-in.html#.create

Reply:
Gotcha, that makes sense 👍